### ISSUE TITLE
 [Bug] Column with missing values is not filled with nan values

### ISSUE THREAD

**Issue by vasili111** [man-group/dtale#184]
In Jupiter lab (I have installed it with `pip install dtale`). Column with missing values is not filled with `nan` values:

```
import pandas as pd
import numpy as np
import dtale

df = pd.DataFrame({'var1': ['a', np.nan, np.nan,np.nan, np.nan],
                   'var2': [np.nan, 'b', np.nan , np.nan, np.nan], 
                   'var3': [np.nan, 'c', 'x', np.nan, np.nan],
                   'var4': [np.nan, np.nan, np.nan, 'd', np.nan],
                   'var5': [np.nan, np.nan , np.nan, np.nan, np.nan]
                 })

dtale.show(df)
```

![image](https://user-images.githubusercontent.com/7338799/79942090-a54d7b00-842b-11ea-8503-9de7f15ffde2.png)

**Comment by aschonfeld** [comment]
@vasili111 so the issue with this is that because you've set all the values for `var5` to `np.nan` pandas will return the data type of that column as a `float64`.  The other column as strings which will return the string value of `np.nan` as "nan".  Whereas for numeric column it will return a. blank string.  Which behavior would you prefer:

1) return the string "nan" in all cases regardless of data type
2) return an empty string regardless of data type

I agree the current behavior is a little odd.

**Comment by aschonfeld** [comment]
I'm just gonna go with filling them all with "nan" strings.  This should be released sometime this weekend
![image](https://user-images.githubusercontent.com/11547371/80257473-0012ec80-864f-11ea-91fa-644be0765699.png)

**Comment by vasili111** [comment]
@aschonfeld 
I agree with you. Thank you.

**Comment by aschonfeld** [comment]
fixed v1.8.10
